examples/java/src/main/java/glide/examples/ClusterExample.java

```diff
@@ -0,0 +1,160 @@
+        GlideClusterClientConfiguration config =
+                GlideClusterClientConfiguration.builder()
+                        .address(
+                                NodeAddress.builder()
+                                        .host(host)
+                                        .port(port1)
+                                        .port(port2)
+                                        .port(port3)
+                                        .port(port4)
+                                        .port(port5)
+                                        .port(port6)
+                                        .build())
+                        .build();
+
+        try {
+            GlideClusterClient client = GlideClusterClient.createClient(config).get();
+            return client;
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Failed to create client: " + e.getMessage());
+        }
+        return null;
+    }
+
+    /**
+     * Executes the main logic of the application, performing basic operations such as SET, GET, PING,
+     * and INFO REPLICATION using the provided GlideClusterClient.
+     *
+     * @param client An instance of <code>GlideClusterClient</code>.
+     */
+    public static void appLogic(GlideClusterClient client) {
+
+        // Send SET and GET
+        try {
+            CompletableFuture<String> setResponse = client.set("foo", "bar");
+            Logger.log(Logger.Level.INFO, "app", "Set response is " + setResponse.get());
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Error during SET: " + e.getMessage());
+        }
+
+        try {
+            CompletableFuture<String> getResponse = client.get("foo");
+            Logger.log(Logger.Level.INFO, "app", "Get response is " + getResponse.get());
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Error during GET: " + e.getMessage());
+        }
+
+        // Send PING to all primaries (according to Valkey's PING request_policy)
+        try {
+            CompletableFuture<String> pong = client.ping();
+            Logger.log(Logger.Level.INFO, "app", "Ping response is " + pong.get());
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Error during PING: " + e.getMessage());
+        }
+
+        // Send INFO REPLICATION with routing option to all nodes
+        try {
+            CompletableFuture<ClusterValue<String>> infoResponse =
+                    client.info(
+                            InfoOptions.builder().section(InfoOptions.Section.REPLICATION).build(), ALL_NODES);
+            Logger.log(
+                    Logger.Level.INFO,
+                    "app",
+                    "INFO REPLICATION responses from all nodes are " + infoResponse.get());
```

Comment:
In Python's cluster example, client.info() doesn't do any extra handling to display the multiple values. I follow Python's implementation, and the above code will only provide the following output which is not very useful:

     app - INFO REPLICATION responses from all nodes are glide.api.models.ClusterValue@130161f7

However, if I change it to `infoResponse.getMultiValue()` or using a for loop (similar to info_with_multi_node_route_and_options in integTest), the output will become very messy and complicated by dumping all info responses. It might not be a suitable for a "basic example". What do you suggest?